Cargo.toml

```diff
@@ -181,3 +181,6 @@ opt-level = 3
 opt-level = 3
 [profile.release.package.base64-simd]
 opt-level = 3
+
+[patch.crates-io]
+deno_graph = { path = "../deno_graph" }
```

Comment:
oooh, TIL...

Reply:
Me too. This is much cleaner than updating everything directly. I'll have to switch my "to local source" scripts to use this.

Reply:
It's nice, but limited. Yesterday I needed to override both `deno_graph` and `deno_ast` (which `deno_graph` depends on) and wasn't able to do that

Reply:
Oh, that sucks then 😂